### DISCUSSION TITLE
What's the name of the feature that moves the editor scroll left/right depending on the position of the cursor?

### DISCUSSION THREAD

**Discussion by seung7642** [JetBrains/ideavim#916] [Q&A] [answered]
Hi, I'm Newby, and I've just decided to use Vim on IntelliJ.

The most inconvenient feature for me is the ability to scroll left/right depending on the position of the cursor. It's hectic.
What is the name of this feature, and how do I disable it?

Thank you.

**Comment by AlexPl292** [comment]
I'm not sure what issue you're referring to. Can you please record a small screencast?

**Reply by seung7642** [reply]
Thank you so much for your quick response. 
The problem has been solved.

**Comment by citizenmatt** [comment, marked as answer]
This is probably [the `'sidescroll'` option](https://vimhelp.org/options.txt.html#%27sidescroll%27). It's the number of columns to scroll when the cursor hits the edge of the screen. By default, it's `0`, which means to scroll so that the cursor is in the middle of the screen (if the line is long enough). You can set this to `1` and it will scroll one column at a time. Also check out [the related `'sidescrolloff'` option](https://vimhelp.org/options.txt.html#%27sidescrolloff%27) which will keep that many columns at the edge of the screen before scrolling.

In IdeaVim 2.12.0, the `'scrolloff'` option will use IntelliJ's own settings, unless it's explicitly set. So you can change this using IntelliJ's own settings - check out the "Scroll Offset" section of the _Settings | Editor | General_ settings page. Or you can set it directly using `:set scrolloff=1` in your `~/.ideavimrc` file.

**Reply by seung7642** [reply]
I disabled the function you talked about and it was resolved. 
Thank you so much. :)
